packages/blockchain-wallet-v4-frontend/src/data/components/nfts/sagas.ts

```diff
@@ -152,7 +152,7 @@ export default ({ api }: { api: APIType }) => {
       // how many token_ids are the same?
       const non_unique_token_ids_map = {}
       let non_unique_token_ids = 0
-      for (let i = 0; i < token_ids.length; i++) {
+      for (let i = 0; i < token_ids.length; i += 1) {
```

Comment:
The gods of clean code love you

Reply:
linter was complaining :P or else i would not touch the `nfts` folder